Issue 7247 raise warning jit (#7476)
inmanta/inmanta-core: docs/install/1-install-server.rst

  # IPv6 local connections:
  host    all             all             ::1/128                 md5

> Make sur JIT is disabled for the psql database as it will impact performances.

Reviewer: ```suggestion
Make sure JIT is disabled for the psql database as it might result in poor query performance.
```

Reviewer: You forgot about this comment I think

Author: indeed sorry. It is changed

> Make sure JIT is disabled for the psql database as it might result in poor query performance.

Reviewer: The comment of Hugo got reverted by applying my suggestion.
```suggestion
Make sure JIT is disabled for the PostgreSQL database as it might result in poor query performance.
```

To disable JIT, set

.. code-block:: text

  # disable JIT
  jit = off

> in ``/etc/postgresql/13/main/postgresql.conf``.

Reviewer: I guess you are mentioning the path to the `postgresql.conf` file, like it is on Ubuntu. On RHEL, this file is present in a different location:
```suggestion
in ``/var/lib/pgsql/13/data/postgresql.conf``.
```

> Restart the PostgreSQL server to apply the changes made in the ``pg_hba.conf`` and  ``postgresql.conf`` files:

Reviewer: `pg_hba.conf`?

Author: I don't know what this file is it was already in the docs I didn't add it

Reviewer: Ah. I see. This is a section with all the PostgreSQL-related config and not only the GIT config.
